### ISSUE TITLE
Enable linting as a pre-commit hook

### ISSUE THREAD

**Issue by davidnixon** [Call-for-Code-for-Racial-Justice/Five-Fifths-Voter#168]
### Background on the problem the feature will solve/improved user experience
Currently linting with prettier `yarn ci-format` has not been applied consistently in the services directory.

### Describe the solution you'd like
1. Lint the current files
2. Force lint as part of pre-commit (see carbon vue repo as an example)
3. This should apply to all dirs including `services` and `ui`

### Tasks 
1. Add pre-commit hooks modeled on carbon/vue repo https://github.com/carbon-design-system/carbon-components-vue
2. Lint all files with `yarn ci-format` and commit
3. Test by updating a file with incorrect spacing or other lint issue and they try to commit it.
4. **BONUS POINTS** Add lint test to pull request so that an error appears if files are not linted.

### Acceptance Criteria 
All files linted and pre-commit hook working.

**Comment by davidnixon** [comment]
I would still like to see this fixed. The https://github.com/davidnixon/carbon-components-vue repo has nice support for this so probably someone could just copy the bits form there.

**Comment by upkarlidder** [comment]
@davidnixon I submitted a PR that uses `yarn run lint`. Would you like that changed to `yarn ci-format`? If so, we can leave this open for hacktoberfest.

@demilolu

**Comment by marcvelasco** [comment]
@upkarlidder i pushed a changes to this in #281, one thing i noted though was that the package.json in both services and UI have a requirement on node 14.x, not sure if this is a hard requirement or not, but any one who has a newer version of node (I had v16.11.1 for example), wouldn't be able to npm install:

npm install
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'services@1.0.0',
npm WARN EBADENGINE   required: { node: '14.x' },
npm WARN EBADENGINE   current: { node: 'v16.11.1', npm: '8.0.0' }
npm WARN EBADENGINE }

**Comment by davidnixon** [comment]
@marcvelasco yep thats the intent. We want to make sure we have a consistent dev environment and currently we are using 14.
I have a bunch of project locally and many require different node version (one ongoing project is stick at node 8 👎 ) So I use nvm which you can [instal](https://github.com/nvm-sh/nvm#installing-and-updating)
And then `nvm use 14`
